docs/sources/awssqs.md

```diff
@@ -1,113 +1,93 @@
-    - [In a Docker container](#in-a-docker-container)
+The event source captures messages sent to a [AWS SQS queue](sqs-docs) and sends them as CloudEvents to an event sink.
 
 ## Prerequisites
 
-* Register an AWS account
-* Create an [Access Key][doc-accesskey] in your AWS IAM dashboard.
-* Create a [SQS queue][doc-sqs].
+### SQS queue
 
-## Deployment to Kubernetes
+If you don't already have an AWS SQS queue, create one by following the instructions in the [Getting started with Amazon SQS][sqs-getting-started] guide.
 
-The _AWS SQS event source_ can be deployed to Kubernetes in different manners:
+### Amazon Resource Name (ARN)
 
-* As an `AWSSQSSource` object, to a cluster where the TriggerMesh _AWS Sources Controller_ is running.
-* As a Knative `ContainerSource`, to any cluster running Knative Eventing.
+A fully qualified ARN is required to uniquely identify the AWS SQS queue.
 
-> :information_source: The sample manifests below reference AWS credentials (Access Key) from a Kubernetes Secret object
-> called `awscreds`. This Secret can be generated with the following command:
->
-> ```console
-> $ kubectl -n <my_namespace> create secret generic awscreds \
->   --from-literal=aws_access_key_id=<my_key_id> \
->   --from-literal=aws_secret_access_key=<my_secret_key>
-> ```
->
-> Alternatively, credentials can be used as literal strings instead of references to Kubernetes Secrets by replacing
-> `valueFrom` attributes with `value` inside API objects' manifests.
+![SQS queue](../images/awssqs-source/sqs-queue.png)
 
-### As a AWSSQSSource object
-
-Copy the sample manifest from `config/samples/awssqssource.yaml` and replace the pre-filled `spec` attributes with the
-values corresponding to your _AWS SQS_ queue. Then, create that `AWSSQSSource` object in your Kubernetes cluster:
+As shown in the above screenshot, you can obtain the ARN of a SQS queue from the AWS console. Alternatively you can also use the [AWS CLI][aws-cli]. The following command retrieves the ARN of a SQS queue named `MyQueue` in the `us-west-2` region.
 
 ```console
-$ kubectl -n <my_namespace> create -f my-awssqssource.yaml
+$ aws --region us-west-2 sqs get-queue-attributes --queue-url $(aws --region us-west-2 sqs list-queues --queue-name MyQueue | jq -r .QueueUrls[0]) --attribute-names QueueArn
+{
+    "Attributes": {
+        "QueueArn": "arn:aws:sqs:us-west-2:043455440429:MyQueue"
+    }
+}
 ```
 
-### As a ContainerSource object
-
-Copy the sample manifest from `config/samples/awssqs-containersource.yaml` and replace the pre-filled environment
-variables under `env` with the values corresponding to your _AWS SQS_ queue. Then, create that `ContainerSource` object
-in your Kubernetes cluster:
+If you don't have the AWS CLI installed on your workstation, use the following template to compose a fully qualified ARN of a SQS queue.
 
-```console
-$ kubectl -n <my_namespace> create -f my-awssqs-containersource.yaml
+```
+arn:aws:sqs:{awsRegion}:{awsAccountId}:{queueName}
 ```
```

Comment:
> If you don't have the AWS CLI installed

Maybe not super relevant for SQS since you showed above that the ARN could be obtained from the console. See my comment at line 17.